pkg/sources/git/git.go

```diff
@@ -84,7 +84,7 @@ func (s *Source) JobID() int64 {
 }
 
 // Init returns an initialized GitHub source.
-func (s *Source) Init(aCtx context.Context, name string, jobId, sourceId int64, verify bool, connection *anypb.Any, concurrency int) error {
+func (s *Source) Init(aCtx context.Context, name string, jobId, sourceId int64, verify bool, connection *anypb.Any, _ int) error {
```

Comment:
I think it would be better to pass `concurrency` along to `NewGit` instead of removing it here.

Reply:
Yea so I had removed it because it wasn't being used in the `NewGit` constructor call, but what you are suggesting is i pass that in as opposed to the current implementation `runtime.NumCPU()`? Also follow up can concurrency be 0? In the even 0 is passed into `Init` can we pass 0 into `NewGit` or should we check that and if it's 0 set it to the `runtime.NumCPU()`